src/cascadia/TerminalApp/Pane.h

```diff
@@ -246,9 +246,9 @@ class Pane : public std::enable_shared_from_this<Pane>
 
     struct ControlEventTokens
     {
-        winrt::event_token _connectionStateChanged{ 0 };
-        winrt::event_token _warningBell{ 0 };
-        winrt::event_token _closeTerminalRequested{ 0 };
+        winrt::Microsoft::Terminal::Control::TermControl::ConnectionStateChanged_revoker _ConnectionStateChanged;

```

Comment:
minor nit: we should avoid `_Uppercase` in new code

Reply:
I have half a mind for some sort of xmacro / `til` template for stuff like this. There's now a collection of places where we're all setting up revokers like this. Like, there's gotta be a better way.... Anywho, this was me being preemptive, which is not always warranted